### DISCUSSION TITLE
schema des couleurs

### DISCUSSION THREAD

**Discussion by hyppocampe1965** [facelessuser/ColorHelper#220] [General]
Bonjour,
je viens d'installer 'colorheper' mais il ne fonctionne pas encore.
Je n'ai pas accès  à 'info color' par exemple.
Je pense sans certitude que cela doit être la configuration du schéma des couleurs ou du choix du thème.
Merci si vous pouvez m'informer

**Comment by facelessuser** [comment]
Google Translation (to English):
> Hello,
I just installed 'colorhelper' but it still doesn't work.
I don't have access to 'info color' for example.
I think without certainty that it must be the configuration of the color scheme or the choice of theme.
Thanks if you can let me know

Unfortunately, my French is very limited, so I'm using Google Translate, please forgive any mistranslations.

I'm more than happy to help you find a solution, but I'll need precise details in order for me to begin. Please try to provide me with a simple reproducible example. In your case, I'll need to know what color scheme you are using, where I can acquire it, and what, if any, custom configurations you've applied.

Google Translation (to French):

> Malheureusement, mon français est très limité, j'utilise donc Google Traduction, veuillez excuser les erreurs de traduction.
>
> Je suis plus qu'heureux de vous aider à trouver une solution, mais j'ai besoin de détails précis pour que je puisse commencer. S'il vous plaît essayez de me fournir un exemple reproductible simple. Dans votre cas, j'aurai besoin de savoir quelle palette de couleurs vous utilisez, où je peux l'acquérir et, le cas échéant, les configurations personnalisées que vous avez appliquées.

**Comment by hyppocampe1965** [comment]
Monsieur Issac Muse,


Thank you for your answer.
Sorry but my english is not good.
So i prefer  to answer in french.

En fait le theme utilisé est sur "auto" et le schéma des couleurs indique que "mariana" est selectionné.

J'ai réalisé que j'étais peut-être dans la  bonne configuration et que j'utilise mal "colorhelper".

Je constate tout de même que en cliquant dans le menu contextuel et en choisissant "colorhelper" , 
"color info" reste grisé .
Mais cela est peut-être normal.
Je vais donc mieux lire la documentation sur "colorherper".

Vous pouvez tout de même de me donner des conseils
car je viens juste d'installer Sublime text , build 4126

Merci à vous
De : "Isaac Muse"
A : "facelessuser/ColorHelper" ,"hyppocampe1965" ,"Author"
Envoyé: samedi 2 Juillet 2022 15:52
Objet : Re: [facelessuser/ColorHelper] schema des couleurs (Discussion #220)
 

 

Google Translation (to English):


Hello,
I just installed 'colorhelper' but it still doesn't work.
I don't have access to 'info color' for example.
I think without certainty that it must be the configuration of the color scheme or the choice of theme.
Thanks if you can let me know


Unfortunately, my French is very limited, so I'm using Google Translate, please forgive any mistranslations.

I'm more than happy to help you find a solution, but I'll need precise details in order for me to begin. Please try to provide me with a simple reproducible example. In your case, I'll need to know what color scheme you are using, where I can acquire it, and what, if any, custom configurations you've applied.

Google Translation (to French):


Malheureusement, mon français est très limité, j'utilise donc Google Traduction, veuillez excuser les erreurs de traduction.

Je suis plus qu'heureux de vous aider à trouver une solution, mais j'ai besoin de détails précis pour que je puisse commencer. S'il vous plaît essayez de me fournir un exemple reproductible simple. Dans votre cas, j'aurai besoin de savoir quelle palette de couleurs vous utilisez, où je peux l'acquérir et, le cas échéant, les configurations personnalisées que vous avez appliquées.

**Reply by facelessuser** [reply]
Google Translate (to English)

> In fact the theme used is on "auto" and the color scheme indicates that "mariana" is selected.
>
> I realized that maybe I was in the right setup and using "colorhelper" wrong.
> 
> I note all the same that by clicking in the context menu and choosing "colorhelper" ,
"color info" remains greyed out.
But this may be normal.
So I'll better read the documentation on "colorherper".
> 
> You can still give me advice
because I just installed Sublime text , build 4126

I am assuming you are stating that you are not getting color previews in your sublime color scheme?

I need you to be very specific. In what type of file are you trying to get color previews? Provide me with the file type, and whatever Sublime syntax file you are using to do syntax highlighting. I cannot answer the question until I fully understand what is being asked.

Google Translate (to French):

>Je suppose que vous déclarez que vous n'obtenez pas d'aperçu des couleurs dans votre sublime palette de couleurs ?
>
> J'ai besoin que vous soyez très précis. Dans quel type de fichier essayez-vous d'obtenir des aperçus de couleur ? Fournissez-moi le type de fichier et le fichier de syntaxe Sublime que vous utilisez pour effectuer la coloration syntaxique. Je ne peux pas répondre à la question tant que je n'ai pas bien compris ce qui est demandé.
